00-introduction.md

```diff
@@ -23,6 +23,62 @@ This is version 0.
 10. [BOLT #10](10-dns-bootstrap.md): DNS Bootstrap and Assisted Node Location
 11. [BOLT #11](11-payment-encoding.md): Invoice Protocol for Lightning Payments
 
+## The Spark: A Short Introduction to Lightning
+
+Lightning is a protocol for fast payments using Bitcoin, using a
+network of channels.
+
+### Channels
+
+Lightning works by establishing *channels*: two participants share
+custody of some bitcoin (e.g. 0.1 bitcoin), spendable only with both
+their signatures.
+
+Initially they each hold a bitcoin transaction which sends all the
+bitcoin (e.g. 0.1) back to one party.  They can sign a new bitcoin
+transaction which splits these funds differently, e.g. 0.09 to one
+party, 0.01 to the other, and invalidate the previous bitcoin
+transaction so it won't be spent.
```

Comment:
won't->can't

Reply:
No: it *can* still be spent, but there's a massive disincentive to do so...